tests/test-update-refspec-checkpoint.c

```diff
@@ -62,7 +61,7 @@ insert_update_refspec_metadata_for_commit (guint         commit,
 
   g_hash_table_insert (*out_metadata_hashtable,
                        GUINT_TO_POINTER (commit),
-                       create_checkpoint_target_metadata (new_refspec));
+                       create_checkpoint_target_metadata (new_ref));
 }
```

Comment:
I think we should retain a test here that it doesn't crash when you give it a refspec with a remote. That said, such a test may not work considering that it uses `g_warning` and I'm pretty sure that gtester sets `G_DEBUG=fatal-warnings`

Reply:
Oh wait, you did that in the test below. Ignore this comment then.